settings.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-# vim: ai ts=4 sts=4 et sw=4 encoding=utf-8
```

Comment:
probably was the encoding. i've noticed that mine has that error occasionally. we should probably just remove all of these from HQ right?